exporter/prometheusremotewriteexporter/config.go

```diff
@@ -48,6 +48,9 @@ type Config struct {
 	// If enabled, all the resource attributes will be converted to metric labels by default.
 	ResourceToTelemetrySettings resourcetotelemetry.Settings `mapstructure:"resource_to_telemetry_conversion"`
 	WAL                         *WALConfig                   `mapstructure:"wal"`
+
+	// Allow disabling the addition of target_info metric
+	DisableTargetInfoMetric bool `mapstructure:"disable_target_info_metric"`
```

Comment:
Is it possible that we would add further customization options for `target_info`? If so I would rework this as:

```
target_info:
  enabled: <true|false>
```

cc @dashpole , not sure if there are specific conventions about this on this exporter

Reply:
I can update this config. Questions: should `enabled` default to true to keep with the existing release or are you ok with defaulting this to false (would it be considered breaking to default to false)?

Reply:
also would you want to update the way it's passed into prwExporter or are you ok to keep that the same?

Reply:
@max-psi updated config. lmk whether you want to prwExporter to also be changed 🙇

Reply:
It should be defaulted to true.